### DISCUSSION TITLE
Keybinding for showing the bookmarks panel?

### DISCUSSION THREAD

**Discussion by jaybonthius** [mbnuqw/sidebery#1130] [General]
Is there a keybinding for showing the bookmarks panel? Not `ctrl-shift-b` to show the toolbar bookmarks—the bookmarks panel in Sidebery.

**Comment by albino1** [comment]
It's just CTRL+B:

![temp](https://github.com/mbnuqw/sidebery/assets/12164957/b5689f08-7f66-4f65-b4b0-5afdfecebfac)

**Reply by jaybonthius** [reply]
Weird, that doesn't do it for me. `ctrl+B` does nothing for me, `⌘+B` opens the Firefox bookmarks sidebar and closes Sidebery. For context I am on the recent beta release [v5.0.0rc4](https://github.com/mbnuqw/sidebery/releases/tag/v5.0.0rc4) on a Mac.

**Reply by albino1** [reply]
I wonder if something has gotten unset for you somehow, since that's just a built in default Firefox shortcut, it doesn't involve Sidebery at all.

Check the `Bookmarks -> Bookmarks Sidebar` section of this page:

https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly

Edit: In the Sidebery Keybindings is an option called `Open/Close sidebar panel`, you could possibly just use that to toggle back to the Sidebery sidebar from Bookmarks.
